pkgs/development/libraries/libxc/default.nix

```diff
@@ -11,6 +11,11 @@ stdenv.mkDerivation rec {
     hash = "sha256-JYhuyW95I7Q0edLIe7H//+ej5vh6MdAGxXjmNxDMuhQ=";
   };
 
+  postPatch = ''
```

Comment:
Could you please put in a comment, stating that is already in the upstream and can be removed with the next update?

Reply:
Done